ignite/distributed/comp_models/xla.py

```diff
@@ -137,19 +137,33 @@ def spawn(  # type: ignore[override]
             "OR": "or",
         }
 
-        def _do_all_reduce(self, tensor: torch.Tensor, op: str = "SUM") -> torch.Tensor:
+        def _do_all_reduce(
+            self, tensor: torch.Tensor, op: str = "SUM", group: Optional[Union[Any, List[int]]] = None
+        ) -> torch.Tensor:
             if op not in self._reduce_op_map:
                 raise ValueError(f"Unsupported reduction operation: '{op}'")
+            if group is not None and not isinstance(group, list):
+                raise ValueError("group should be list of int or list of list of int")
+            elif group is not None and not all(isinstance(item, list) for item in group):
+                raise ValueError("group should be list of int or list of list of int")
+            elif group is not None and not all(isinstance(item, int) for list_ in group for item in list_):
```

Comment:
This line occurs errors, because `int` in `List[int]` is not iterable.
How can this be solved?

Reply:
> How can this be solved?

Try to figure out what's wrong here and find a appropriate solution :)